src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/Preview.js

```diff
@@ -294,6 +317,19 @@ class Preview extends React.Component<Props> {
                                     value={this.previewStore && this.previewStore.targetGroup}
                                 />
                             }
+                            {this.segments.length > 0 &&
```

Comment:
That part is still something I am not completely happy with. The `Preview` already has to know about `webspaces`, `segments` and `targetGroups`, and it is not really extensible. I still think we should make use of the `ToolbarConfigStore` here, that one would be easier to extend than the current approach.